com.unity.netcode.gameobjects/Runtime/Messaging/MessageQueue/MessageQueueContainer.cs

```diff
@@ -44,6 +44,27 @@ public enum MessageQueueProcessingTypes
             Receive,
         }
 
+        static Dictionary<int, string> s_MessageTypeNames;
```

Comment:
I don't think this is a big deal. But... when running multi-instance, those static will be shared across instances. So, if there were a bug/issue with filling this dictionary, our testing would not be able to detect it.

I thought we were trying to avoid all static. Is there anyway around it ?

Reply:
I think this instance is fine because it's building an immutable data set based on compile-time types - it's not state, just a fast lookup. multiple instances should be able to access the data without issue and the data should never change

Reply:
one thing I can do to make this "safer" is change it to a read only dictionary though